tests/actions/patient.test.js

```diff
@@ -0,0 +1,67 @@
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import {
+  fetchPatientRecord,
+  fetchPatientNote,
+} from '../../app/js/actions/patient';
+
+import {
+  SET_PATIENT, SET_NOTE
+} from '../../app/js/actions/actionTypes';
+
+import mockData from '../../__mocks__/mockData';
+import locationMock from '../../__mocks__/locationMock';
+
+const middlewares = [thunk];
+
+const mockStore = configureMockStore(middlewares);
+window.location = locationMock;
+
+const applicationDistribution = window.location.href.split('/')[3];
```

Comment:
Is the above the same as what you some times case appBaseUrl?

Reply:
No, the apiBaseUrl was generated off the applicationDistribution
const apiBaseUrl = `/${applicationDistribution}/ws/rest/v1`;

Reply:
What does applicationDistribution mean? Can you give examples of possible values that it can take on?

Reply:
Yes, applicationDistribution refers to the what the app core is running on.
`openmrs-standalone` could be a typical example

Reply:
Wouldn't context path be a better name? Just google it a bit.

Reply:
Thanks, will update accordingly

Reply:
After some digging as suggested, I realised contextPath or applicationContext will do, so I choose contextPath, thanks for pointing this out